rector.php

```diff
@@ -8,7 +8,7 @@
 use Rector\Core\ValueObject\PhpVersion;
 use Rector\Php80\Rector\Class_\StringableForToStringRector;
 use Rector\Php80\Rector\ClassMethod\FinalPrivateToPrivateVisibilityRector;
-use Rector\Php80\Rector\ClassMethod\OptionalParametersAfterRequiredRector;
+use Rector\CodeQuality\Rector\ClassMethod\OptionalParametersAfterRequiredRector;
```

Comment:
@eliseacornejo will this change work if by any chance rector 0.12.4 is installed? I would upgrade the minimum rector version specified in `composer.json` to `^0.12.17`

Reply:
Yes, you are totally right, this is not going to work on 0.12.4 I will do the change!